assets/map_style.txt

```diff
@@ -0,0 +1,209 @@
+[
```

Comment:
Same as above

Reply:
At the end is an asset, makes no sense do a complete folder 2 files